assets/source/setup-guide/app/components/HealthCheck/index.js

```diff
@@ -167,7 +167,7 @@ const HealthCheck = () => {
 		pending_initial_configuration: {
 			status: 'warning',
 			message: __(
-				'The feed is being configured. Depending on the number of products this may take a while as the feed needs to be fully generated before its been sent to Pinterest for registration. You can check the the status of the generation process in the Catalog tab.',
+				'The feed is being configured. Depending on the number of products this may take a while as the feed needs to be fully generated before its been sent to Pinterest for registration. You can check the status of the generation process in the Catalog tab.',
```

Comment:
```suggestion
				'The feed is being configured. Depending on the number of products this may take a while as the feed needs to be fully generated before it is sent to Pinterest for registration. You can check the status of the generation process in the Catalog tab.',
```